atlassian_jwt_auth/key.py

```diff
@@ -97,21 +97,22 @@ def load(self, issuer):
 class FilePrivateKeyRetriever(object):
 
     def __init__(self, private_key_repository_path):
-        self.private_key_repository = FilePrivateKeyRepository(
-            private_key_repository_path)
+        self.repository = FilePrivateKeyRepository(private_key_repository_path)
 
     def load(self, issuer):
         key_identifier = self._find_first_key_id(issuer)
-        private_key_pem = self.private_key_repository.load_key(key_identifier)
+        private_key_pem = self.repository.load_key(key_identifier)
         return key_identifier, private_key_pem
 
     def _find_first_key_id(self, issuer):
-        for key_identifier in self.private_key_repository.find_valid_key_id(
-                issuer):
+        key_identifier = None
+        for key_identifier in self.repository.find_valid_key_ids(issuer):
```

Comment:
I renamed the method to `_find_last_key_id`, hopefully it's clearer now what it's doing.